plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -171,10 +171,12 @@ pluginOptions =
               , Implication (== True) posPreserveLogging True
               , Implication (== True) posCaseOfCaseConservative True
               , Implication (== True) posInlineConstants False
+              , Implication (== True) posDoSimplifierEvaluateBuiltins False
               , Implication (== False) posRelaxedFloatin True
               , Implication (== False) posPreserveLogging False
               , Implication (== False) posCaseOfCaseConservative False
               , Implication (== False) posInlineConstants True
+              , Implication (== False) posDoSimplifierEvaluateBuiltins True
```

Comment:
My OCD would prefer `id` over `(== True)` and `not` over `(== False)`, but I guess it's currently more readable.